docs/userguide/declarative_config.rst

```diff
@@ -170,9 +170,9 @@ Special directives:
       project directory (i.e. the directory containing ``setup.cfg``/``pyproject.toml``).
 
   .. attention::
-      When using the ``file:`` directive, please make sure that all necessary
-      files are included in the ``sdist``. You can do that via ``MANIFEST.in``
-      or using plugins such as ``setuptools-scm``.
+      For versions prior to <VERSION>: When using the ``file:`` directive,
```

Comment:
Let me know if this language should be changed. Stating the version that ships the change will be useful.